packages/ant-database/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
isn't the global .editorconfig enough? I thought that you don't need them in sub directories if nothing changes